lib/msf/ui/tip.rb

```diff
@@ -0,0 +1,38 @@
+# -*- coding: binary -*-
+# frozen_string_literal: true
+
+module Msf
+  module Ui
+    ###
+    #
+    # Module that contains some most excellent tips.
+    #
+    ###
+    module Tip
+      def self.highlight(string)
```

Comment:
Will this factor in @adamgalway-r7's work eventually?

Reply:
I'm working on a highlighting function for searching through a string and replacing a pattern with a background of a chosen color and am adding it into Rex-Text. @adfoster-r7 could use that but for this simple use case it'd probably be overkill